### PR TITLE
bump to 1.1.3

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <vieux@docker.com>

### CODE REVIEW

**Comment by vieux** [comment]
ping @dongluochen @nishanttotla

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.1.3 (2016-02-24)
+
+- Fix missing HostConfig for rescheduled containers
+- Fix TCP connections leak
+- Support `docker run --net <node>/<netwotk> ...`
```
Comment (nishanttotla) [inline CHANGELOG.md]: typo: `netwotk` -> `network`

**Comment by dongluochen** [comment]
LGTM

**Comment by chanwit** [comment]
Other than some typos, LGTM

**Comment by nishanttotla** [comment]
LGTM

**Comment by nishanttotla** [comment]
@dongluochen this is not related to the PR, but the following test is failing. Any idea why?
```
not ok 151 refresh loop detects failure
# (in test file nodemanagement/nodehealth.bats, line 62)
#   `[ "$status" -eq 0 ]' failed
```

**Pull request merged**